app/controllers/courses_controller.rb

```diff
@@ -572,6 +572,7 @@ def write_cuds(cuds)
                                                   grade_policy: new_cud[:grade_policy],
                                                   lecture: new_cud[:lecture])
         existing.assign_attributes(params.permit(:lecture, :section, :grade_policy))
+        existing.dropped = false
```

Comment:
Could you quickly explain what is happening by setting dropped = false? I assume this is for when dropped students are added back, they should become "undropped"

Reply:
Yep, that's correct! When we parse a "red" student, existing.dropped gets set to true, but previously we never set it back to false when the student gets added back in.